lib/explorer/polars_backend/lazy_frame.ex

```diff
@@ -7,16 +7,27 @@ defmodule Explorer.PolarsBackend.LazyFrame do
   alias Explorer.PolarsBackend.Native
   alias Explorer.PolarsBackend.Shared
   alias Explorer.PolarsBackend.DataFrame, as: Eager
-  alias Explorer.PolarsBackend.LazyFrame, as: PolarsLazyFrame
 
   alias FSS.Local
   alias FSS.S3
 
   import Explorer.PolarsBackend.Expression, only: [to_expr: 1, alias_expr: 2]
 
-  defstruct resource: nil
+  defstruct frame: nil, stack: []
```

Comment:
I wonder if there is a way to keep it as `defstruct resource: nil, stack: []` and we tell Rustler to not serialize or bother with the `stack: []` field?

Reply:
I don't think it's possible to do easily today. I will add this to my list and revisit in the near future.